vue-frontend/src/pages/jobs/apply/[id].vue

```diff
@@ -366,28 +361,38 @@ export default {
     }),
   },
   methods: {
+    async setJob() {
+      if (this.job == null || this.job.id != this.id) {
+        await this.getCareerDetails(true);
+      }
+      this.setCareerDetails();
+    },
     async getCareerDetails() {
```

Comment:
Why we haven't added `setDetails` flag here?

Reply:
Good find!